# Force cmake to use the EB values for everything
# wpoely86@gmail.com
diff -ur pytorch-0.3.1.orig/torch/lib/build_libs.sh pytorch-0.3.1/torch/lib/build_libs.sh
--- pytorch-0.3.1.orig/torch/lib/build_libs.sh	2018-02-09 18:07:43.000000000 +0100
+++ pytorch-0.3.1/torch/lib/build_libs.sh	2018-04-12 20:15:15.242733426 +0200
@@ -70,8 +70,8 @@
   ${CMAKE_VERSION} ../../$1 -DCMAKE_MODULE_PATH="$BASE_DIR/cmake/FindCUDA" \
               -DTorch_FOUND="1" \
               -DCMAKE_INSTALL_PREFIX="$INSTALL_DIR" \
-              -DCMAKE_C_FLAGS="$BUILD_C_FLAGS" \
-              -DCMAKE_CXX_FLAGS="$BUILD_C_FLAGS $CPP_FLAGS" \
+              -DCMAKE_C_FLAGS="$BUILD_C_FLAGS $CFLAGS" \
+              -DCMAKE_CXX_FLAGS="$BUILD_C_FLAGS $CPP_FLAGS $CXXFLAGS" \
               -DCMAKE_EXE_LINKER_FLAGS="$LDFLAGS" \
               -DCMAKE_SHARED_LINKER_FLAGS="$LDFLAGS" \
               -DCMAKE_INSTALL_LIBDIR="$INSTALL_DIR/lib" \
@@ -96,6 +96,7 @@
               -Dnanopb_BUILD_GENERATOR=0 \
               -DCMAKE_DEBUG_POSTFIX="" \
               -DCMAKE_BUILD_TYPE=$([ $DEBUG ] && echo Debug || echo Release) \
+              -DCMAKE_VERBOSE_MAKEFILE=ON \
               ${@:2}
   make install -j$(getconf _NPROCESSORS_ONLN)
   cd ../..
